### DISCUSSION TITLE
Profile for Raise3D E2

### DISCUSSION THREAD

**Discussion by DrCWood** [OrcaSlicer/OrcaSlicer#1516] [General]
Hello! Could someone please create a profile for the Raise3D E2 with 0.4mm and 0.6mm nozzles? I've become hooked on using OrcaSlicer with my BambuLabs printers and would love to use it on the E2 as well.

**Comment by mcavuto** [comment]
Building on this, I think it would be great if someone could create a profile for the Raise3D Pro 3 with the Hyperspeed Upgrade kit.  (The kit greatly improves the speed of the printer and makes it far more usable.) Thank you in advance if anyone feels like doing this!

**Comment by mr-j-tree** [comment]
I would love a profile for the Raise3D E2 as well!

**Comment by mmcnair91** [comment]
Has anyone done this? I found this but i think this requires you to do the duet update https://www.proschi3d.de/raise-3d-e2-umbau
